src/ducks/transactions/TransactionCategoryEditor.jsx

```diff
@@ -9,32 +9,54 @@ import CategoryChoice from 'ducks/categories/CategoryChoice'
 /**
  * Edits a transaction's category through CategoryChoice
  */
-const TransactionCategoryEditor = props => {
+const TransactionCategoryEditor = ({
+  transactions,
+  beforeUpdate,
+  afterUpdate,
+  afterUpdates,
+  onCancel
+}) => {
   const client = useClient()
-  const { transaction, beforeUpdate, afterUpdate, onCancel } = props
+  const makeCategoryId = useMemo(() => getCategoryId(transactions[0]), [
```

Comment:
makeCategoryId looks like the name of a function, but here it is a value. categoryId seems to be better here.

Reply:
Exact j'ai eu la même réflexion, j'avais souvenir que categoryId était déjà pris via les props (d'où le `make`)... mais en fait non... par contre oui `make` est plus pour une fonction qu'une valeur, pas pertinent ici.